query-service-impl/src/main/java/org/hypertrace/core/query/service/prometheus/PromqlQuery.java

```diff
@@ -0,0 +1,33 @@
+package org.hypertrace.core.query.service.prometheus;
+
+import java.util.List;
+
+public class PromqlQuery {
+
+  private final List<String> queries;
+  private final boolean isInstantRequest;
+  private final long startTimeMs;
+  private final long endTimeMs;
+  private final long stepMs;
+
+  public PromqlQuery(
+      List<String> queries,
+      boolean isInstantRequest,
+      long startTimeMs,
```

Comment:
Should this take a query time range object?

Reply:
Updated